ontopy/excelparser.py

```diff
@@ -9,18 +9,23 @@
 
 Note that correct case is mandatory.
 """
-import sys
+from typing import Tuple, Union, Sequence
 import warnings
-from typing import Tuple, Union
-import pyparsing
+
 import pandas as pd
+import pyparsing
+
 import ontopy
-from ontopy import World, get_ontology
+from ontopy import get_ontology
 from ontopy.utils import NoSuchLabelError
 from ontopy.manchester import evaluate
 import owlready2  # pylint: disable=C0411
 
 
+class ExcelError(Exception):
```

Comment:
```suggestion
class ExcelError(EMMOntoPyException):
```

Reply:
Good suggestion. Committed separately. Thanks. 

Wouldn't `EMMOntoPyError` be a better name instead of `EMMOntoPyException`?

Reply:
I think EMMOntoPyException makes sense to use since it inherits from Exception. Alternatively ExcelError could be renamed to ExcelException?